daemon/config/config.go

```diff
@@ -53,7 +52,11 @@ const (
 	DefaultContainersNamespace = "moby"
 	// DefaultPluginNamespace is the name of the default containerd namespace used for plugins.
 	DefaultPluginNamespace = "plugins.moby"
-
+	// DefaultMinAPIVersion is the minimum API version supported by the API.
+	// This version can be overridden through the "DOCKER_MIN_API_VERSION"
+	// environment variable. The minimum allowed version is determined
+	// by [github.com/docker/docker/api.MinVersion].
```

Comment:
oh! this reference can be removed now

Reply:
maybe I should move the "set version" handling to the config package; let me have a peek what that looks like